### DISCUSSION TITLE
Dokku App and DB Persistent Storage?

### DISCUSSION THREAD

**Discussion by ibnsina77** [dokku/dokku#4604] [Q&A]
Does anybody know, how to have Dokku App + Dokku DB (like Postgres) mounted externally? How sohould I do it.

Should I create app, create DB, link them together, and then?

Should I go with storage:mount db (or app)?

**Comment by Cellane** [comment]
Do you mean that you wish to have the storage used by Postgres in an external location?
I haven’t tried that personally and not sure if it would work (and especially how performant/laggy it would be), but I’d personally create the app, create the DB, link them together, stop both, move the database’s storage folder to the desired external storage, put a symbolic link or mount to the original location to point to the new location and start app and database back up. No idea how well it would work though.

**Reply by ibnsina77** [reply]
Thank you @Cellane .

Yes, exactly. But I really do not have Idea how to move databes's storage outside and having link with the app? That is actually my question.
